hub/publish.go

```diff
@@ -77,6 +71,24 @@ func (h *Hub) PublishHandler(w http.ResponseWriter, r *http.Request) {
 		panic(err)
 	}
 
+	h.metrics.NewUpdate(topics)
```

Comment:
Can you move that after `io.WriteString(w, u.ID)` and the log (to reduce the latency).

Reply:
:heavy_check_mark: